src/develop/develop.c

```diff
@@ -1531,6 +1531,18 @@ static gboolean _dev_auto_apply_presets(dt_develop_t *dev)
     g_list_free_full(iop_list, free);
     dt_ioppr_set_default_iop_order(dev, imgid);
   }
+  else if(is_display_referred || is_scene_referred)
+  {
+    // we have no auto-apply order, so apply iop order, depending of the worflow
+    GList *iop_list;
+    if(is_display_referred)
```

Comment:
and test if(is_scene_referred) here to use DT_IOP_ORDER_V30 and in all other cases (display referred or none) use the DT_IOP_ORDER_LEGACY order. I mean the iop order is not optional, we need one so even when no workflow is set we can apply it here. Or am I missing something ?

Reply:
oh, I thought that one order was applied in any case actually, so what done is more a correction than anything.
But you're right, we can handle all case here (remove this if above) as a safety check.
Now the question is : what should be the default order if workflow is set to none ? I would say v3.0 as it's more a "power user" value, but...

Reply:
Agreed, none -> v3.0